WordPress/Classes/Services/BlogService.m

```diff
@@ -831,7 +830,15 @@ - (void)updateMultiAuthor:(BOOL)isMultiAuthor forBlog:(NSManagedObjectID *)blogO
         if (!blog) {
             return;
         }
-        blog.isMultiAuthor = isMultiAuthor;
+        blog.isMultiAuthor = users.count > 1;
+        if (blog.username != nil) {
+            for (RemoteUser *user in users) {
+                if ([user.username isEqualToString:blog.username]) {
```

Comment:
The users response won't contain wp.com usernames for a self hosted site, or did you mean something else?